### PR TITLE
polish for cloud social provider SSH use

### PR DESCRIPTION
**Pull request by trevj**
Applying some polish, based on what I learned while working on the installer:
https://github.com/uProxy/uproxy-lib/pull/349

Main two things:
 * reject with `{message: 'xxx'}` objects (for freedom)
 * remove some handlers on the SSH objects which don't actually get called in practise

[Review on Reviewable](https://reviewable.io/reviews/uproxy/uproxy-lib/350)

### CODE REVIEW

**Comment by trevj** [comment]
BTW, I hope the promise rejection fixes may help with this issue:
https://github.com/uProxy/uproxy/issues/2183

File: src/cloud/social/provider.ts [hunk 1]
```diff
@@ -26,6 +26,9 @@ const STORAGE_KEY = 'cloud-social-contacts';
 const ADMIN_USERNAME = 'giver';
 const REGULAR_USERNAME = 'getter';
 
+// Timeout for establishing an SSH connection.
+const CONNECT_TIMEOUT_MS = 10000;
```
Comment (soycode) [inline src/cloud/social/provider.ts]: It looks like the default is 20000, just curious the reason for shortening it.

Comment (trevj) [inline src/cloud/social/provider.ts]: 20s just seemed like a long time to be sitting there waiting for an error.

Comment (soycode) [inline src/cloud/social/provider.ts]: :+1: fair enough.

**Comment by soycode** [comment]
:+1:

**Pull request merged**